triagebot.toml

```diff
@@ -417,6 +417,48 @@ message_on_remove = "Issue #{number}'s prioritization request has been removed."
 message_on_close = "Issue #{number} has been closed while requested for prioritization."
 message_on_reopen = "Issue #{number} has been reopened."
 
+[notify-zulip."beta-nominated"]
```

Comment:
Contrary to rust-lang/rust#82689's self-proclaimed *hack*, rustdoc is now *occupying* the labels https://github.com/rust-lang/rust/labels/beta-nominated and https://github.com/rust-lang/rust/labels/stable-nominated. Swapping them around to make https://github.com/rust-lang/rust/labels/T-rustdoc the primary label would not allow us to register a hook for both https://github.com/rust-lang/rust/labels/beta-nominated **and** https://github.com/rust-lang/rust/labels/stable-nominated.

Reply:
I could add a FIXME comment here noting this and suggesting patching `triagebot` to support “`[[notify-zulip."beta-nominated"]]`” etc. (i.e., allowing users to specify an array of actions instead of just one), do nothing or implement that patch right away, at your liking!

Reply:
I've added FIXMEs.